### PR TITLE
4 site logo

### LINKED ISSUE #4: SiteLogo
## Component Type:  Stateless

Check each box once you have completed done the task.

-   [ ] Component created
-   [ ] Styles written
-   [ ] Storybook created, with stories for all normal states
-   [ ] Tests written, with snapshot test at minimum
-   [ ] Code reviewed

Add PR link here when ready:

### PR DESCRIPTION
**Pull request by S-unya**
**Ensure each of these tasks is complete before merging**

Use component name as PR title and request code review before merging. You can
open a PR and request review for an incomplete component if you'd like.

-   [x] Component created
-   [x] Styles written
-   [x] Storybook created, with stories for all normal states
-   [x] Tests written, with snapshot test at minimum

Closes #4

### CODE REVIEW

**Comment by S-unya** [comment]
@ascorbic, I'd be very interested in your typescript expertise regarding how to express the rest/spread of the Props in Button

File: src/components/Button/Button.tsx [hunk 1]
```diff
@@ -5,19 +5,34 @@ import * as React from "react";
...
 import * as styles from "./Button.module.scss";
 
+interface ElProps {
+    style?: React.CSSProperties;
+    className?: string;
+    arrow?: boolean;
+    alternate?: boolean;
+}
+
 interface Props {
```
Comment (ascorbic) [inline src/components/Button/Button.tsx]:
```
interface Props extends React.HTMLAttributes<HTMLButtonElement>
```

Comment (S-unya) [reply src/components/Button/Button.tsx]: Thanks.

**Comment by S-unya** [comment]
This is ready now

**Comment by S-unya** [comment]
@ascorbic, this work is done in #26 where I need to make a LinkButton component. I'm hoping to get this through and then sort out the conflicts once through

**Pull request merged**